.ci/checker-framework.groovy

```diff
@@ -75,26 +68,36 @@ private static Set<String> getCheckerFrameworkProfiles() {
  * the suppressed errors.
  *
  * @param profile the checker framework profile to execute
- * @param flag command line argument flag to determine output format
  * @return {@code 0} if checker framework report is as expected, {@code 1} otherwise
  */
-private static int checkCheckerFrameworkReport(String profile, String flag) {
+private static int checkCheckerFrameworkReport(String profile) {
     final XmlParser xmlParser = new XmlParser()
     final String suppressedErrorsFileUri =
             ".${File.separator}.ci${File.separator}" +
                     "checker-framework-suppressions${File.separator}${profile}-suppressions.xml"
     final List<String> checkerFrameworkErrors = getCheckerFrameworkErrors(profile)
-    Set<CheckerFrameworkError> errors = Collections.emptySet()
+    List<CheckerFrameworkError> errors = Collections.emptyList()
     if (!checkerFrameworkErrors.isEmpty()) {
         errors = getErrorFromText(checkerFrameworkErrors)
     }
     final File suppressionFile = new File(suppressedErrorsFileUri)
-    Set<CheckerFrameworkError> suppressedErrors = Collections.emptySet()
+    List<CheckerFrameworkError> suppressedErrors = Collections.emptyList()
     if (suppressionFile.exists()) {
         final Node suppressedErrorsNode = xmlParser.parse(suppressedErrorsFileUri)
         suppressedErrors = getSuppressedErrors(suppressedErrorsNode)
     }
-    return compareErrors(errors, suppressedErrors, flag)
+
+    String newSuppresion = '<?xml version="1.0" encoding="UTF-8"?>\n<suppressedErrors>'
+    errors.each {
+        newSuppresion += it.toXmlString() + '\n'
+    }
+    newSuppresion += "</suppressedErrors>\n"
```

Comment:
Can we print empty line if `errors` is empty so start/end on separate lines (similar to our {}s on ifs)

Reply:
done